GTG/gtk/editor/editor.py

```diff
@@ -571,20 +571,33 @@ def inserttag_clicked(self, widget):
             self.textview.insert_text(" @", itera)
 
     def on_parent_select(self, widget):
+        parent_box = Gtk.Box.new(Gtk.Orientation.VERTICAL, 0)
+        parent_button = self.builder.get_object("parent")
         parents = self.task.get_parents()
+
         if len(parents) == 1:
-            self.vmanager.open_task(parents[0])
-        elif len(self.task.get_parents()) > 1:
-            '''
-            TODO (jakubbrindza): If there is more than 1 parent,open a popover
-            with their names and allow choosing which one to open.
-            '''
-            pass
+            self.open_parent(None, parents[0])
+        elif len(parents) > 1:
+            for parent in parents:
+                parent_name = self.req.get_task(parent).get_title()
+                button = Gtk.ToolButton.new(None, parent_name)
+                button.connect("clicked", self.open_parent, parent)
+                parent_box.add(button)
+
+            self.parent_popover = Gtk.Popover.new(parent_button)
+            self.parent_popover.add(parent_box)
+            self.parent_popover.set_property("border-width", 0)
+            self.parent_popover.set_position(Gtk.PositionType.BOTTOM)
+            self.parent_popover.set_transitions_enabled(True)
+            self.parent_popover.show_all()
```

Comment:
Can you move all this into Gtk code into a separate method, something like self.show_multiple_parent_popover(parents)